ReactViewControl/IViewModule.cs

```diff
@@ -4,14 +4,18 @@ namespace ReactViewControl {
 
     public interface IViewModule {
 
-        string JavascriptSource { get; }
+        string MainSource { get; }
+
+        string OriginalSourceFolder { get; }
 
         string NativeObjectName { get; }
 
         string Name { get; }
 
         string Source { get; }
 
+        string[] ExternalSources { get; }
```

Comment:
ExternalSources ... don't like External... how about DependencySources or ... you name it